src/route.js

```diff
@@ -9,19 +9,43 @@ import QueryExplorer from './views/QueryExplorer.vue';
 import Log from './views/Log.vue';
 import User from './views/User.vue';
 
-Vue.use(VueRouter);
-
-var router = new VueRouter({
+const router = new VueRouter({
   routes: [
-    { path: '/',                        component: Home },
-    { path: '/activity/:host',          component: Activity },
-    { path: '/activity/:host/:date',    component: Activity },
-    { path: '/buckets',                 component: Buckets },
-    { path: '/buckets/:id',             component: Bucket },
-    { path: '/query',                   component: QueryExplorer },
-    { path: '/log',                     component: Log },
-    { path: '/u/:username',             component: User },
-  ]
+    {
```

Comment:
Not really an improvement, but looks good enough and conforms to the standard so it's fine.

Reply:
Yeah, the old one also seems a bit better than the newer one, but I guess it yielded an error.